music/app/controllers/services_controller.rb

```diff
@@ -1,23 +1,28 @@
 class ServicesController < ApplicationController
   def new
+    @congregations = Congregation.all
     if params[:last]
       @last = Service.find_by(date: Date.parse(params[:last]))
-      @resource = OpenStruct.new(date: @last.date + 7, body: "")
+      @resource = OpenStruct.new(date: @last.date + 7, body: "", congregation_id: params[:congregation_id])
```

Comment:
should this be `params[:resource][:congregation_id]`? Similarly in other places in this file.